.halcyon/sandbox-extra-apps

```diff
@@ -0,0 +1,3 @@
+alex-3.1.3
+happy-1.19.4
+yesod-bin-1.4.3.1
```

Comment:
It's nice that there's no contamination. In the case that the extra apps and my app work across the same set of dependencies (which has been the case so far but may not be likely in general), it'd be nice to only have to manage one file. The ones you liked are small though, so oh well.